### DISCUSSION TITLE
Estados de issues que requieren varias iteraciones para completarse

### DISCUSSION THREAD

**Discussion by angelalvaigle** [Arquisoft/faq#59] [Q&A]
Los issues que no tienen un fin claro, sino que habrá que hacer varias iteraciones para completarlos (como p.ej. los relativos a elaboración de documentación), ¿se pasan a "done" y se vuelven a sacar a "in progress" las veces que sea necesario?, ¿se pasan a "done" y luego se crean issues nuevos?, ¿o se dejan en "in progress" hasta cerrarlos definitivamente?

**Comment by labra** [comment]
No hay ninguna norma concreta sobre el tema, así que podéis hacer un poco lo que consideréis adecuado. En muchos proyectos es habitual tener _issues_ abiertos durante mucho tiempo y personalmente creo que tiene sentido. 

En realidad la nomenclatura de _done_ o _in progress_ es algo que depende de qué metodología estéis adoptando como equipo y hasta qué punto estáis siguiendo esa metodología de forma estricta, y en principio, en la asignatura no hemos puesto ninguna restricción sobre metodología a emplear, así que podéis utilizar la que consideréis mejor.
